conf/php-fpm.conf

```diff
@@ -0,0 +1,430 @@
+; Start a new pool named 'www'.
```

Comment:
Why did you remove `conf/extra_php-fpm.conf ` and add back the template?

When we use `usage` and `footprint` parameters, like in the install script:
```
 ynh_add_fpm_config --usage=$usage --footprint=medium --package="$extra_php_dependencies"
```
we don't need this template, it's fully generated from the [default one](https://github.com/YunoHost/yunohost/blob/1279755463100e6c4a0ba7d4b50f8f39658c8eac/data/helpers.d/php#L165-L214)

Reply:
I can' recall the exact alert but with `conf/extra_php-fpm.conf`,  php7.3 service was failling and with the template it was working...
I'll give it another try and report the exact output error.

Reply:
well... I can't reproduce my "php7.3 service was failling" error...